service-node/backend/src/main/scala/org/enterprisedlt/fabric/service/node/rest/JsonRestEndpoint.scala

```diff
@@ -90,6 +109,44 @@ class JsonRestEndpoint(
         }
     }
 
+
+    private def createUploadHandler(o: AnyRef, m: Method): (HttpServletRequest, HttpServletResponse) => Unit = {
+        logger.info(s"Creating handler for: ${m.getName}")
+        val eClazz = classOf[Either[String, Any]]
+        m.getReturnType match {
+            case x if x.isAssignableFrom(eClazz) =>
```

Comment:
additional check for method parameters require: must be array of one parameter with the right type (refer to method declaration)

Reply:
added.